compiler-rt/lib/mpk_untrusted/alloc_site_handler.cpp

```diff
@@ -1,6 +1,12 @@
 #include "alloc_site_handler.h"
 
+extern "C" {
+  bool is_safe_address(void* addr) { return false; }
+}
```

Comment:
Isn't this going to cause problems? this isn't a weak symbol, unless I've missed something, so I would imagine this would always fail to link against our allocator.

Reply:
oh, found the weak decl. bad grep on my part